app/helpers/report_helper.rb

```diff
@@ -1,5 +1,6 @@
 module ReportHelper
   include_concern 'Editor'
+  include DashboardWidgetHelper
```

Comment:
I'm concerned that if we add these helpers here, then you could get naming conflicts when other PRs start introducing them also such as in #8345 .  I'm curious if you actually need to include then or not.  Let's discuss.

Reply:
I renamed them to `ReportDashboardWidgetHelper`